### DISCUSSION TITLE
What are the changes in 0.2.2?

### DISCUSSION THREAD

**Discussion by Unpuertomex** [alexkn/node-red-contrib-dlna#18] [Q&A] [answered]
What were the changes going from 0.2.1--->0.2.2. The logs do not reflect changes other than compatibility, not that this is a problem, but wondering if I'm overlooking something important.

### [0.2.1](https://github.com/alexkn/node-red-contrib-dlna/compare/v0.2.0...v0.2.1) (2021-01-31)

**Comment by alexkn** [comment, marked as answer]
Only updated dependencies due to security issues

**Reply by Unpuertomex** [reply]
Thanks
